app/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,7 @@
 	<string name="blink_slow">Slow</string>
 	<string name="blink_normal">Normal</string>
 	<string name="blink_frenetic">Frenetic</string>
+
+	<string name="share">Share ...</string>
+	<string name="send_to">Send using</string>
```

Comment:
That latter one I just forgot. It is leftover from testing another method of sharing.